### PR TITLE
Date time col name

### PR DESCRIPTION
**Pull request by boshek**
`pm_annual_caaqs` didn't seem to like values for `dt` that aren't `date_time`. This PR is meant to fix that and add a test. Here is the bug reproduced:

``` r
library(rcaaqs)
library(dplyr)
#> 
#> Attaching package: 'dplyr'
#> The following objects are masked from 'package:stats':
#> 
#>     filter, lag
#> The following objects are masked from 'package:base':
#> 
#>     intersect, setdiff, setequal, union

pm_annual_caaqs(pm25_sample_data, by = c("ems_id", "site"))
#> Calculating PM 2.5 daily average
#> Calculating PM 2.5 annual average
#> Calculating PM 2.5 annual CAAQS metric
#> 
#> CAAQS results for PM2.5 annual 
#>   * Includes CAAQS ambient results only
#> --------------------------------------
#> 4 elements:
#>       * daily_avg ( 10901 rows x 8 columns )
#>       * yearly_avg ( 30 rows x 14 columns )
#>       * three_yr_rolling ( 30 rows x 21 columns )
#>       * caaqs ( 30 rows x 12 columns )
#> Access them with the get_ functions (see ?get_caaqs)

pm25_sample_data %>%
  rename(DATE_TIME = date_time) %>%
  pm_annual_caaqs(dt = "DATE_TIME", by = c("ems_id", "site"))
#> Calculating PM 2.5 daily average
#> Error: arrange() failed at implicit mutate() step. 
#> x Could not create a temporary column for `..3`.
#> i `..3` is `date_time`.
```

### CODE REVIEW

**Comment by ateucher** [comment]
Good catch, thanks @boshek! Travis is broken I think, and I haven't set up GH Actions - assuming it passes locally for you I'll take my chances and merge it now if you think it's okay?

**Review by ateucher — APPROVED** [review]

**Comment by boshek** [comment]
@ateucher - fixed a couple other minor things. Check still complaining about a couple minor issues but they aren't show stoppers.

File: tests/testthat/test-1_utils.R [hunk 1]
```diff
@@ -44,7 +44,7 @@ test_that("check_class gets appropriate match", {
 
 test_that("initial_check catches hourly problems", {
   
-  d <- d[-c(100:110),]
+  d <- dplyr::slice(d, -c(100:110))
```
Comment (ateucher) [inline tests/testthat/test-1_utils.R:47]: Why this change?

**Review by ateucher — APPROVED** [review]
